src/Api.js

```diff
@@ -0,0 +1,28 @@
+const BASE_URL = "https://pokeapi.co/api/v2/";
+
+async function callApi(endpoint, options = {}) {
+  const opts = options;
+  opts.headers = {
```

Comment:
If you are gonna use `headers` like this, this could be a constant too.

Reply:
already changed, thanks for the comment